panic on tab-completion of "s3://"

Hitting tab in an attempt to autocomplete a blank `s3://` url produces a panic:

```
% fasts3 ls s3://panic: MissingRegion: could not find region configuration

goroutine 1 [running]:
panic(0x417200, 0xc420010980)
    /usr/local/go/src/runtime/panic.go:500 +0x1a1
main.main()
    /Users/tyler/Documents/gohome/src/github.com/TuneOSS/fasts3/main.go:215 +0x64c
panic: MissingRegion: could not find region configuration

goroutine 1 [running]:
panic(0x417200, 0xc420010980)
    /usr/local/go/src/runtime/panic.go:500 +0x1a1
main.main()
    /Users/tyler/Documents/gohome/src/github.com/TuneOSS/fasts3/main.go:215 +0x64c
```

--- a/main.go
+++ b/main.go
@@ -249,7 +249,8 @@ func main() {
 	})
 
 	if err != nil {
-		panic(err)
+		fmt.Fprintf(os.Stderr, "Encountered an error: %s\n", err)
+		return
 	}
 
 	svc := s3.New(awsSession, aws.NewConfig())
@@ -259,24 +260,28 @@ func main() {
 	case ls.FullCommand():
 		listCh, err := Ls(svc, *lsS3Uris, *lsRecurse, *lsDelimiter, *lsSearchDepth, lsKeyRegex)
 		if err != nil {
-			panic(err)
+			fmt.Fprintf(os.Stderr, "Encountered an error: %s\n", err)
+			return
 		}
 		printLs(listCh, *lsHumanReadable, *lsWithDate)
 
 	case stream.FullCommand():
 		err := Stream(svc, *streamS3Uris, *streamDelimiter, *streamSearchDepth, *streamIncludeKeyName, streamKeyRegex, *streamOrdered, *streamRaw)
 		if err != nil {
-			panic(err)
+			fmt.Fprintf(os.Stderr, "Encountered an error: %s\n", err)
+			return
 		}
 	case get.FullCommand():
 		Get(svc, *getS3Uris, *getRecurse, *getDelimiter, *getSearchDepth, getKeyRegex, *getSkipExisting)
 	case cp.FullCommand():
 		if err := Cp(svc, *cpS3Uris, *cpRecurse, *cpDelimiter, *cpSearchDepth, cpKeyRegex, *cpFlat); err != nil {
-			panic(err)
+			fmt.Fprintf(os.Stderr, "Encountered an error: %s\n", err)
+			return
 		}
 	case rm.FullCommand():
 		if err := Rm(svc, *rmS3Uris, *rmRecurse, *rmDelimiter, *rmSearchDepth, rmKeyRegex); err != nil {
-			panic(err)
+			fmt.Fprintf(os.Stderr, "Encountered an error: %s\n", err)
+			return
 		}
 	}
 }

